specification/columns/skupricedetails.md

```diff
@@ -1,22 +1,22 @@
 # SKU Price Details
 
-The **SKU Price Details** column represents a list of relevant properties shared by all charges with the same [**SKU Price ID**](#skupriceid). These properties provide qualitative and quantitative details about the service represented by a **SKU Price ID**. This can enable practitioners to calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. These properties can also help a practitioner understand the specifics of a **SKU Price ID** and differentiate it other **SKU Price IDs**.
+The SKU Price Details column represents a list of relevant properties shared by all charges with the same [SKU Price ID](#skupriceid). These properties provide qualitative and quantitative details about the service represented by a SKU Price ID. This can enable practitioners to calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. These properties can also help a practitioner understand the specifics of a SKU Price ID and differentiate it other SKU Price IDs.
 
-The *SkuPriceDetails* column adheres to the following requirements:
+The SkuPriceDetails column adheres to the following requirements:
 
-* The *SkuPriceDetails* column MUST be in [*KeyValueFormat*](#key-valueformat).
+* The SkuPriceDetails column MUST be in [KeyValueFormat](#key-valueformat).
```

Comment:
I suggest using the attribute name instead of the attribute ID when referencing formatting attributes, at least for now, since this is consistent with what we've used in previous releases and the majority of columns.

```suggestion
* The SkuPriceDetails column MUST be in [Key-Value Format](#key-valueformat).
```